pyfar/coordinates.py

```diff
@@ -665,27 +611,16 @@ def systems(self, show='all', brief=False):
 
         Parameters
         ----------
-        show: string
+        show: string, optional
             'current' to list the current corrdinate system or 'all' to list
-            all coordinate systems. The default is 'all'
-        brief : boolean
+            all coordinate systems. The default is 'all'.
+        brief : boolean, optional
             Will only list the domains, conventions and units if True. The
             default is False.
 
         Returns
         -------
         Prints to console.
-
```

Comment:
Put `Notes` here instead `.. note::` of above?

Reply:
`Notes` is a section from the numpy docstring guide that I used it things that are nice to know or for things that are specific only for certain parameters or returns . `.. note::` can be used to highlight things that are important for general use. I would suggest to keep both.

Reply:
Thanks for clarifying, removing `.. note::` at other positions made it more consitent